report_aging.php

```diff
@@ -54,13 +54,19 @@ function GetDeviceByAge($year) {
             $previous_year = $year - 1;
             $selectSQL = sprintf("SELECT DeviceID, Label, Owner, MfgDate, PrimaryContact FROM fac_Device WHERE (DATEDIFF(NOW(), (CASE WHEN MfgDate>'1969-12-31' THEN MfgDate ELSE InstallDate END))/365)<%d AND (DATEDIFF(NOW(), (CASE WHEN MfgDate>'1969-12-31' THEN MfgDate ELSE InstallDate END))/365)>=%d ORDER BY Owner, MfgDate ASC, Label", $year, $previous_year);
             foreach ($dbh->query($selectSQL) as $deviceRow) {
-                $deviceList[$deviceRow['DeviceID']] = Device::RowToObject($deviceRow);
```

Comment:
@$deviceList[$deviceRow['DeviceID']] = Device::RowToObject($deviceRow);

I've tested just suppressing the errors and changing the select to * and both return a valid report with no spam in the error log.  Neither has any effect on the content of the report.